build.sbt

```diff
@@ -10,6 +10,7 @@ val silencerVersion = "1.7.0"
 
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory, SbtWeb)
+  .disablePlugins(JUnitXmlReportPlugin)
```

Comment:
Is this to remove that crappy alert?

Reply:
according to some confluence this should fix the error on jenkins where test results cannot be read